sql/moz-fx-data-shared-prod/telemetry_derived/newtab_clients_daily_v1/query.sql

```diff
@@ -0,0 +1,135 @@
+WITH visits_data AS (
+  SELECT
+    client_id,
+    submission_date,
+    ANY_VALUE(legacy_telemetry_client_id) AS legacy_telemetry_client_id,
+    COUNT(DISTINCT newtab_visit_id) AS newtab_visit_count,
+    ANY_VALUE(normalized_os) AS normalized_os,
+    ANY_VALUE(normalized_os_version) AS normalized_os_version,
+    ANY_VALUE(country_code) AS country_code,
+    ANY_VALUE(locale) AS locale,
+    ANY_VALUE(channel) AS channel,
+    ANY_VALUE(browser_version) AS browser_version,
+    ANY_VALUE(browser_name) AS browser_name,
+    ANY_VALUE(default_search_engine) AS default_search_engine,
+    ANY_VALUE(default_private_search_engine) AS default_private_search_engine,
+    MAX(pocket_is_signed_in) AS pocket_is_signed_in,
+    MIN(pocket_enabled) AS pocket_enabled,
+    MIN(pocket_sponsored_stories_enabled) AS pocket_sponsored_stories_enabled,
+    MIN(topsites_enabled) AS topsites_enabled,
+    ANY_VALUE(newtab_homepage_category) AS newtab_homepage_category,
+    ANY_VALUE(newtab_newtab_category) AS newtab_newtab_category,
+    ANY_VALUE(topsites_rows) AS topsites_rows,
+    ANY_VALUE(experiments) AS experiments,
+    SUM(CASE WHEN had_non_impression_engagement THEN 1 ELSE 0 END) AS had_non_impression_engagement,
```

Comment:
Is this meant to be a number? Something like:
```suggestion
    SUM(CASE WHEN had_non_impression_engagement THEN 1 ELSE 0 END) AS visits_with_non_impression_engagement,
```
Similarly below with `had_non_search_engagement`

Reply:
Thanks, incorporated these